R/plumb-block.R

```diff
@@ -304,7 +304,12 @@ evaluateBlock <- function(srcref, file, expr, envir, addEndpoint, addFilter, pr)
         stopOnLine(lineNum, file[lineNum], e)
       })
       if (is.function(func)) {
-        func(pr)
+        assign("__plump_block_check__", TRUE, envir = envir)
+        on.exit(rm("__plump_block_check__", envir = envir), add = TRUE)
+        fpr <- func(pr)
+        if (inherits(fpr, "Plumber") && !isTRUE(fpr$environment[["__plump_block_check__"]])) {
```

Comment:
If we start with a `.`, then it won't show up with `ls()` by default.

```suggestion
        assign(".__plump_block_check__", TRUE, envir = envir)
        on.exit(rm(".__plump_block_check__", envir = envir), add = TRUE)
        fpr <- func(pr)
        if (inherits(fpr, "Plumber") && !isTRUE(fpr$environment[[".__plump_block_check__"]])) {
```